inst/crunch-test.R

```diff
@@ -9,6 +9,11 @@ skip_locally <- function (...) {
 }
 
 loadCube <- function (filename) {
+    file <- sub('.*\\/', '', filename)
+    file <- system.file(paste0("cubes/", file), package = "crunch")
```

Comment:
You should just be able to pass `filename` to `system.file`, it calls `file.path` inside.